resources/dw/schema_cluster.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2024 Cloudera. All Rights Reserved.
+//
+// This file is licensed under the Apache License Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, either express or implied. Refer to the License for the specific
+// permissions and limitations governing your use of the file.
+
+package dw
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/booldefault"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var dwAwsClusterSchema = schema.Schema{
```

Comment:
I still do not get why we need a separate file for these schemas. 

I think you might remember my comment on your previous PR in which I reviewed the test case in which you've used this variable for assertion. If this `dwAwsClusterSchema` (or similar) would not exist, the falsy test case the with the assertion would not exist either.

Reply:
you mean like this?

Reply:
yes like this